### PR TITLE
OKD: ironic-rhcos-downloader ocp image can be used for okd

### PR DESCRIPTION
**Pull request by sherine-k**

### CODE REVIEW

**Comment by LorbusChris** [comment]
/hold

@sherine-k the excludeTag for mirroring `ironic-machine-os-downloader` has to be removed here as well

**Comment by sherine-k** [comment]
> /hold
> 
> @sherine-k the excludeTag for mirroring `ironic-machine-os-downloader` has to be removed here as well

Done

File: core-services/release-controller/_releases/release-ocp-4.12-ci.json [hunk 1]
```diff
@@ -71,6 +70,7 @@
           "ironic-inspector",
           "ironic-ipa-downloader",
           "ironic-machine-os-downloader",
+          "ironic-static-ip-manager",
```
Comment (LorbusChris) [inline core-services/release-controller/_releases/release-ocp-4.12-ci.json:73]: I think the addition of this line is not right here

File: core-services/release-controller/_releases/release-ocp-4.13-ci.json [hunk 2]
```diff
@@ -72,6 +71,7 @@
           "ironic-inspector",
           "ironic-ipa-downloader",
           "ironic-machine-os-downloader",
+          "ironic-static-ip-manager",
```
Comment (LorbusChris) [inline core-services/release-controller/_releases/release-ocp-4.13-ci.json:74]: same here

**Review by LorbusChris — CHANGES_REQUESTED** [review]

File: core-services/release-controller/_releases/release-ocp-4.12-ci.json (hunk 1 above)
Comment (LorbusChris) [reply core-services/release-controller/_releases/release-ocp-4.12-ci.json:73]: Instead, the line above should be removed here. There are two configs in this file, which should both be changed in the same way. (remove `ironic-machine-os-downloader` from the excludeTags)

File: ci-operator/step-registry/rosa/credentials/hypershift/mgmt/rosa-credentials-hypershift-mgmt-ref.yaml [hunk 3]
```diff
@@ -16,4 +16,4 @@ ref:
     documentation: The environment for rosa login. The supported values are [production, staging].
 
   documentation: |-
-    Using rosa cli to config the htpasswd idp and grant cluster-admin to the user on a rosa cluster.
+    Using ocm cli to get the kubeconfig of the management cluster.
```
Comment (LorbusChris) [inline ci-operator/step-registry/rosa/credentials/hypershift/mgmt/rosa-credentials-hypershift-mgmt-ref.yaml:19]: @sherine-k looks like this change snuck in during rebase

File: ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh [hunk 4]
```diff
@@ -93,9 +93,9 @@ if [[ -z "$OPENSHIFT_VERSION" ]]; then
   exit 1
 fi
 
-if [[ "$CHANNEL_GROUP" != "stable" ]]; then
-  OPENSHIFT_VERSION="${OPENSHIFT_VERSION}-${CHANNEL_GROUP}"
-fi
+# if [[ "$CHANNEL_GROUP" != "stable" ]]; then
+#   OPENSHIFT_VERSION="${OPENSHIFT_VERSION}-${CHANNEL_GROUP}"
+# fi
```
Comment (LorbusChris) [inline ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh:98]: so does this

**Pull request merged**